Framework/Algorithms/test/PolarizationCorrectionWildesTest.h

```diff
@@ -413,8 +390,6 @@ class PolarizationCorrectionWildesTest : public CxxTest::TestSuite {
 
   void test_SpinStateAddedToSampleLogWhenRequestedNoAnalyser() {
     constexpr size_t numClones{2};
-    BinEdges edges{0.3, 0.6, 0.9, 1.2};
-    const double yVal = 2.3;
     Counts counts{yVal, yVal, yVal};
```

Comment:
```suggestion
    Counts counts(3, yVal);
```
When the value is repeated, you can initialize the counts using this constructor.